### DISCUSSION TITLE
Issues Install M.I.B in GEM 2017 Audi A4 B9

### DISCUSSION THREAD

**Discussion by bmpp22** [Mr-MIBonk/M.I.B._More-Incredible-Bash#267] [Q&A] [answered]
Hi, I am an experienced I.T consultant, but new to M.I.B & Audi REM/GEM.
I am using the mibsolution.one files for my 2017 Audi A4 B9 Sedan Australia
My recently purchased 2017 A4 has SD1,SIM,SD2 in the glove box and 2 x USB data in the armrest tray. It has MMI and CarPlay from the factory, but the maps are still 2016!
I successfully upgraded the Audi original firmware:
from: MHI2_ER_AUG22_S2148_MU0442
to: MHI2_ER_AUG22_K2161_MU1421
I then installed M.I.B v3.6.0 with the K2161 patch files embedded
I can access REM & GEM (4.12s) normally, however after multiple attempts, there is no M.I.B sub menu in GEM, just the white Production menu.
I have repeated the M.I.B install manually and automatically using brand new SanDisk Extreme 170Mb SDcards, but I am seeing no change.
The MMI is working perfectly in the car with firmware at K2161, as is CarPlay and all the normal car functions, Driving Assistance, Parking Assistance, HUD, VC, MMI and navigation.
My goal was to install M.I.B and then install the latest NavDB and maps but I can go no further without access to the M.I.B menu in GEM.
I also retried installing with M.I.B v3.5.1, but still didn't have a M.I.B menu in GEM.
I considered re-flashing K2161 again, but after re-preparing a new SDcard and launching the update, everything was marked as N or NA and the "Start Update" option was greyed out.
On further reading, I saw that I could enable "User Defined" in GEM and then re-flash K2161, but then I wasn't sure what the various module options should be and didn't want to break my working car, so I didn't go any further.
I just don't understand why M.I.B is not installing correctly and display in GEM and any assistance would be appreciated. Thanks Brad.

**Comment by harman-f** [comment]
M.I.B SD card is still in unit?

**Reply by bmpp22** [reply]
Unfortunately not, but I appreciate the feedback. As mentioned I tried both the auto-install off the SD, as well as the manual launch from SD. Perhaps if there were log files for the install (on the SD?) I might be able to garner some more information?

**Comment by harman-f** [comment, marked as answer]
SD card has to stay in unit. No SD no M.I.B

**Reply by bmpp22** [reply]
Thanks for the follow up! Apologies, I didn't understand.
Okay, so you are saying the SD MUST stay in the unit for M.I.B to appear in GEM, even after the install?

**Reply by harman-f** [reply]
Exactly. 
install is just a symlink on unit.

**Reply by bmpp22** [reply]
Thank you mate. I. Did see the “don’t remove SD” in the install directions, but I was thinking more “during” than post.
Entering GEM with the M.I.B SD in slot 1 gives me the M.I.B menu 🙏

**Comment by fr1z05** [comment]
Hey! I have a 2018 Audi A5 with MHI2Q_ER_AUG22_P5092 MU: 1329 updated by me and I also installed M.I.B v3.6.0 with the patch files embedded.
I have the exact same problem as you. I can open GEM but I don t have m.i.b. only production. I installed MIB with a FAT32 USB DRIVE that I never took out after but still only the "production" in GEM. Is it mandatory to use a SD card? Updating the software and installing MIB worked just fine with a USB drive.

**Reply by harman-f** [reply]
Please share log file from SD

**Reply by fr1z05** [reply]
[MU1329-JTB-07414.03.18116603C3.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11990399/MU1329-JTB-07414.03.18116603C3.log)

**Reply by harman-f** [reply]
Did you delete parts of the log?

**Reply by fr1z05** [reply]
No.

**Reply by harman-f** [reply]
Try a different SD card.

**Reply by fr1z05** [reply]
New SD card worked! Thank you!

**Reply by fr1z05** [reply]
But how do I enable the code for updating maps?

**Reply by harman-f** [reply]
PATCH AIO in M.I.B GEM

**Reply by fr1z05** [reply]
Ty

**Comment by lllmaglll** [comment]
I have successfully installed the firmware update for my Audit TT model 2016 to MHI2_ER_AU43X_P5098_MU1339.
I then successfully installed the newest MIB update. 
Unfortunately, there is no REM function for MIB in the REM menu.
The SD card has remained plugged in. 

Do you have any idea what the problem could be?

**Reply by mvprajapati** [reply]
> I have successfully installed the firmware update for my Audit TT model 2016 to MHI2_ER_AU43X_P5098_MU1339. I then successfully installed the newest MIB update. Unfortunately, there is no REM function for MIB in the REM menu. The SD card has remained plugged in.
> 
> Do you have any idea what the problem could be?

Hi Mate,
I've got same issue as you. 
2018 Audi Q5 - Australia.
Updated the firmware to MHI2Q_ER_AUG22_P5092, MU Software: 1329
Turned on developer mode from OBD11 for GEM.
Installed MIB 3.6.0 QR edition from REM.
I've kept SD card inserted. But, same, I am not seeing MIB in GEM. 
Please someone can help? I want to install new maps, and for that need to add "ee" system key.

Cheers,

**Reply by mvprajapati** [reply]
Hi All,
This one resolved.
I was using microSD card with SD card Adaptor. Switched to a complete SD card, and it worked. Got the MIB on GEM.

Thank you.

**Comment by Babu408** [comment]
Hello guys, how are I lost power at some point, I now my system is initializing half way and off and starting again, any help please? 
![Uploading 20250202_095635.jpg…]()

**Comment by Babu408** [comment]
Hello guys, I lost power at some point, and now my system is initializing half way and off and starting again, any help please?
![Uploading 20250131_112516.jpg…]()

**Comment by giohouse** [comment]
Hi guys, i have change something but i do not remeber what.
Now my GEM text size is smaller than standard, the text is very small.
Anyone have ideas?
Thank's a lot

**Comment by Babu408** [comment]
![20250620_195217](https://github.com/user-attachments/assets/09f0483b-7e4e-4b82-b53e-02305f37aa92)
Try this

**Reply by giohouse** [reply]
![image](https://github.com/user-attachments/assets/21404033-d94c-430d-9150-ea0b2a905a3a)
I set like this but don’t have any changes.

**Reply by Babu408** [reply]
Do both

**Reply by Babu408** [reply]
The one below it too.

**Reply by giohouse** [reply]
I've set all parameters like your photo.
Correct?
Did you see my image?

**Reply by Babu408** [reply]
Set Byte 2 like byte 1

**Reply by giohouse** [reply]
It works!!!!
Thanks a lot man!

**Reply by Babu408** [reply]
Pleasure, me I learn the hard way. Now I need your help!

**Reply by giohouse** [reply]
Tell me, i try to help you.
I have another problem with navigation, that tell me "Data Navigation Error".
Tell me what is your problem?

**Reply by Babu408** [reply]
I lost settings for parking. 
![20250714_192305](https://github.com/user-attachments/assets/03738b7b-9404-49a5-b522-560c8ffc4b8a)
Please share.

**Reply by giohouse** [reply]
![image](https://github.com/user-attachments/assets/f9e77d17-3fb8-4f6c-8ecf-b7aaeaff3887)
There are mine

**Reply by Babu408** [reply]
Thanks will try.

**Reply by giohouse** [reply]
![image](https://github.com/user-attachments/assets/d61c4030-f9c6-4c18-88f2-f40812c0d2d7)
Do you have some ideas for this error?

**Reply by giohouse** [reply]
@Babu408

**Reply by Babu408** [reply]
No idea for that one
